src/components/UploaderForm.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import UploaderForm from './UploaderForm';
+import { InternalApiGateway } from '../gateways/internal-api';
+
+test('it renders an uploader panel and a continue button', async () => {
+  const gateway = new InternalApiGateway();
+  const documentTypes = await gateway.getDocumentTypes(true);
+  const evidenceRequest = await gateway.getEvidenceRequest(true);
```

Comment:
Rather than using the gateway (which before long will make real API requests) can you just import the fixture and construct a new evidence request manually?

e.g. 
```js
import evidenceRequestFixture from '../../test/fixture/evidence-request-resonse.json'
import EvidenceRequest from '../domain/evidence-request'
// ...
const evidenceRequest = new EvidenceRequest(evidenceRequestFixture[0])
```

(and same for documentType)

Reply:
Just noticed, you'd have to use the `ResponseMapper` class to map the fixture, instead of instantiating directly, like this:
```js
const evidenceRequest = ResponseMapper.mapEvidenceRequest(evidenceRequestFixture[0])
```

Reply:
Just checking you saw this—we shouldn't be using the InternalApiGateway in tests to generate data, as that class will be making real API requests. Instead use the mock JSON data we already have in the repo as above to build the models you need for the tests.

Reply:
it isn't using that, is it?

it's using the fixture file and response mapper directly.

Reply:
Somehow I ended up looking at an old version of this file 🤦